operator/internal/manifests/storage/options.go

```diff
@@ -106,3 +106,12 @@ type CloudCredentials struct {
 func (o OpenShiftOptions) TokenCCOAuthEnabled() bool {
 	return o.CloudCredentials.SecretName != "" && o.CloudCredentials.SHA1 != ""
 }
+
+func (o Options) AllowStructuredMetadata() bool {
+	for _, schema := range o.Schemas {
+		if schema.Version == lokiv1.ObjectStorageSchemaV13 {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
Small comment WDYT of flipping the logic so this function is future-proof? I'm assuming future versions of the storage schema will support structured metadata